src/scripts/GetChromeDriverVersion.js

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const request = require('sync-request');
+
+/*
+Getting chrome version driver from google
+*/
+const getChromeDriverVersion = () => {
+  const res = request('GET', 'https://chromedriver.storage.googleapis.com/LATEST_RELEASE');
+  console.log('Getting chrome driver version', res.getBody().toString());
+  return res.getBody().toString();
```

Comment:
how about :
const responseString = request('GET', 'https://chromedriver.storage.googleapis.com/LATEST_RELEASE').getBody().toString();
this will remove a need of to string conversion